### DISCUSSION TITLE
Unable to TOR proxy

### DISCUSSION THREAD

**Discussion by GookiGrums** [FreeTubeApp/FreeTube#4125] [Q&A] [answered]
Hello, I'm having trouble connecting through Tor.
I am using the .exe version freetube for windows 10. It works but not with Tor.
I have Downloaded Tor, installed, and am running it. I am using all the default options.
I launch tor and connect (I can use youtube.com on the browser), However, when I connect I get a black bar message that reads "Error getting network information. Is your proxy configured properly?"
If I try to look for a video I get an error "TypeError: Failed to fetch".
When I downloaded Tor, I just got the default download. I did not get the "expert" version as per the site's instructions, I wonder if that's what I did wrong?

I included and image showing youtube on tor, and the error message on freetube with proxy enabled.
https://ibb.co/LxB9mgC

Thank You for any help.

**Comment by patheticcockroach** [comment, marked as answer]
Someone mentioned this question in #4108 

The first thing I'd check would be the port that Tor actually listens to. For some reason, for me it was port 9150 and not 9050 as the default FT value suggests.
You can check it in Tor Browser settings, at the very bottom of the "Connection" tab (about:preferences#connection): "View Logs". Then look for a line like "Opening Socks listener on 127.0.0.1:9150"

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
@GookiGrums
